pkg/sql/codegen/common.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2020 The SQLFlow Authors. All rights reserved.
```

Comment:
The meaning of function `GenerateFeatureColumnCode` is clear, so I would suggest renaming `common.go` to another name, like: `featurecolumngen.go` or something else.

Reply:
Done. `common.go` has been moved to `codegen_feature_column.go` in .https://github.com/sql-machine-learning/sqlflow/pull/2402 .